addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/onboard-account.cfn.yml

```diff
@@ -281,13 +282,64 @@ Resources:
               - ec2:DescribeSubnets
               - ec2:DescribeVpcs
               - ec2:DescribeNetworkInterfaces
+              - ec2:DescribeSecurityGroups
+              - ec2:DescribeAvailabilityZones
+              - ec2:DescribeAccountAttributes
+              - ec2:CreateTags
+              - ec2:DeleteTags
             Resource: '*' # For the actions listed above IAM does not support resource-level permissions and requires all resources to be chosen
           - Effect: Allow
             Action:
               - ssm:GetParameter
             Resource:
               - !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/*'
-
+          - Effect: Allow
+            Action:
+              - ec2:CreateSubnet
+              - ec2:DeleteSubnet
+              - ec2:AssociateSubnetCidrBlock
+              - ec2:DisassociateSubnetCidrBlock
+            Resource: '*'
```

Comment:
Can the Resource be further [filtered](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html) down using condition keys?

Reply:
Made changes to restrict the resources